### DISCUSSION TITLE
Allow export of tabluar report result data to CSV / Excel

### DISCUSSION THREAD

**Discussion by cmrusna97** [cortezaproject/corteza-server#1566] [Ideas]
Request that the capability to export data from table report display elements to CSV / XLSX format be added.

Because of the ability to place multiple display elements in a single report, each individual display element should have a button allowing for the export of its data.

**Comment by cmrusna97** [comment]
This is a critical issue for organizations looking for an alternative to SalesForce.  We need to be able to get report data out of Corteza and into spreadsheet format for various reasons (e.g. sending it to outside activities) without having to get technical about it.

**Reply by kelanik8** [reply]
Hello @cmrusna97,
This would be coming out in a future release here is a logged [github issue](https://github.com/cortezaproject/corteza/issues/1724)
